bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetAuxiliaryHandler.java

```diff
@@ -0,0 +1,160 @@
+                }
+            } else {
+                logger.warn("Unsupported ChannelUID {}", channel);
+            }
+        }
+    }
+
+    @Override
+    protected void requestChannelState(ChannelUID channel) {
+        logger.debug("requestChannelState() thingUID={} channel={}", thing.getUID(), channel.getId());
+        requestStatus(channel.getId());
+    }
+
+    /** helper method to request auxiliary status based on channel */
+    private void requestStatus(String channelId) {
+        Where w = deviceWhere;
+        if (w != null) {
+            try {
+                Response res = send(Auxiliary.requestStatus(toWhere(channelId)));
+                if (res != null && res.isSuccess()) {
+                    ThingStatus ts = getThing().getStatus();
+                    if (ThingStatus.ONLINE != ts && ThingStatus.REMOVING != ts && ThingStatus.REMOVED != ts) {
+                        updateStatus(ThingStatus.ONLINE);
+                    }
+                }
+            } catch (OWNException e) {
+                logger.warn("requestStatus() Exception while requesting auxiliary state: {}", e.getMessage());
+            }
+        } else {
+            logger.warn("Could not requestStatus(): deviceWhere is null");
+        }
+    }
+
+    @Override
+    protected void refreshDevice(boolean refreshAll) {
+        OpenWebNetBridgeHandler brH = bridgeHandler;
+        if (brH != null) {
+            if (brH.isBusGateway() && refreshAll) {
+                long now = System.currentTimeMillis();
+                if (now - lastAllDevicesRefreshTS > ALL_DEVICES_REFRESH_INTERVAL_MSEC) {
+                    try {
+                        send(Auxiliary.requestStatus(WhereAuxiliary.GENERAL.value()));
+                        lastAllDevicesRefreshTS = now;
+                    } catch (OWNException e) {
+                        logger.warn("Exception while requesting all devices refresh: {}", e.getMessage());
+                    }
+                } else {
+                    logger.debug("Refresh all devices just sent...");
+                }
+            }
+        }
+    }
+
+    @Override
+    protected Where buildBusWhere(String wStr) throws IllegalArgumentException {
+        return new WhereAuxiliary(wStr);
+    }
+
+    @Override
+    protected String ownIdPrefix() {
+        return Who.AUX.value().toString();
+    }
+
+    /**
+     * @param channelId the channelId string
+     * @return a WHERE address string based on channelId string
+     */
+    @Nullable
+    private String toWhere(String channelId) {
+        Where w = deviceWhere;
+        if (w != null) {
+            OpenWebNetBridgeHandler brH = bridgeHandler;
+            if (brH != null) {
+                return w.value();
+            } else if (channelId.equals(CHANNEL_SWITCH)) {
+                return w.value();
+            }
+        }
+        return null;
+    }
```

Comment:
This method like it's written does not anything useful. It's better to remove it and just check deviceWhere is not null when it is used (like in requestStatus()).

Reply:
fixed